lib/qa/authorities/linked_data/rdf_helper.rb

```diff
@@ -24,70 +27,6 @@ def init_consolidated_hash(consolidated_results, uri, id)
           consolidated_hash
         end
 
-        def get_linked_data(url)
-          begin
-            graph = RDF::Graph.load(url)
-          rescue IOError => e
-            process_error(e, url)
-          end
-          graph
-        end
-
-        def process_error(e, url)
-          uri = URI(url)
-          raise RDF::FormatError, "Unknown RDF format of results returned by #{uri}. (RDF::FormatError)  You may need to include gem 'linkeddata'." if e.is_a? RDF::FormatError
-          response_code = ioerror_code(e)
-          case response_code
-          when '404'
-            raise Qa::TermNotFound, "#{uri} Not Found - Term may not exist at LOD Authority. (HTTPNotFound - 404)"
-          when '500'
-            raise Qa::ServiceError, "#{uri.hostname} on port #{uri.port} is not responding.  Try again later. (HTTPServerError - 500)"
-          when '503'
-            raise Qa::ServiceUnavailable, "#{uri.hostname} on port #{uri.port} is not responding.  Try again later. (HTTPServiceUnavailable - 503)"
-          else
-            raise Qa::ServiceError, "Unknown error for #{uri.hostname} on port #{uri.port}.  Try again later. (Cause - #{e.message})"
-          end
-        end
-
-        def ioerror_code(e)
-          msg = e.message
-          return 'format' if msg.start_with? "Unknown RDF format"
-          a = msg.size - 4
-          z = msg.size - 2
-          msg[a..z]
-        end
-
-        # Filter a graph to the specified languages
-        # @param [RDF::Graph] the graph to be filtered.
-        # @param [String | Symbol | Array<String|Symbol>] language for filtering graph (e.g. "en" or :en or ["en", "fr"] or [:en, :fr])
-        # @returns [RDF::Graph] graph of linked data filtered on the specified languages
-        def filter_language(graph, language)
-          language = normalize_language(language)
-          return graph if language.nil?
-          graph.each do |st|
-            graph.delete(st) unless !st.object.respond_to?(:language) || st.object.language.nil? || language.include?(st.object.language)
-          end
-          graph
-        end
-
-        # Filter a graph to remove any statement with a blanknode for the subject
-        # @param [RDF::Graph] the graph to be filtered.
-        # @returns [RDF::Graph] graph of linked data with blanknodes removed
-        def filter_out_blanknodes(graph)
```

Comment:
Moved to rdf_service and renamed filter_out_subject_blanknodes to be more precise.